src/Cache.php

```diff
@@ -0,0 +1,31 @@
+<?php declare(strict_types=1);
+
+namespace IW\PHPUnit\DbFixtures;
+
+class Cache
+{
+    private static $instance;
+    private static $data = [];
```

Comment:
change to non-static property. It's not necessary and it's a lot slower to use static one

Reply:
Done